en/04_Changelogs/5.0.0.md

```diff
@@ -227,21 +290,20 @@ their aliases (applied by setting a string key for the array) were being ignored
-- silverstripe/graphql v3 is no longer supported. It is recommended that you ensure you are using graphl v4 in your Silverstripe CMS 4 project before upgrading to Silverstripe CMS 5. There is [documentation for the upgrade process here](https://docs.silverstripe.org/en/4/upgrading/upgrading_to_graphql_4/).
+- Various Symfony dependencies have been upgraded from 4.x to 6.x. A small number of code changes were made in `silverstripe/framework` to work with the symfony 6.x dependencies.
+- `silverstripe/graphql` v3 is no longer supported. It is recommended that you ensure you are using `silverstripe/graphl` v4 in your Silverstripe CMS 4 project before upgrading to Silverstripe CMS 5. There is [documentation for the upgrade process here](https://docs.silverstripe.org/en/4/upgrading/upgrading_to_graphql_4/).
 - PHPUnit 5.7 is no longer supported. It is recommended that you ensure your tests are running with PHPUnit 9 in your Silverstripe CMS 4 project before upgrading to Silverstripe CMS 5. There is [documentation for the upgrade process here](https://docs.silverstripe.org/en/4/upgrading/phpunit9/).
 - `masterminds/html5` is now used to parse HTML content instead of `DomDocument::loadHTML()`. This may cause slight changes in rendered content if the content being saved contains invalid HTML. The optional module `silverstripe/html5` is no longer required for rendering HTML 5 content.
 
 ### TinyMCE upgraded from 4 to 6 {#tinymce6}
 
-TinyMCE has been upgraded up two major versions from 4 to 6. The API for [HTMLEditorConfig](api:SilverStripe\Forms\HTMLEditor\HTMLEditorConfig) and [TinyMCEConfig](api:SilverStripe\Forms\HTMLEditor\TinyMCEConfig) haven't changed at all, but there have been some changes to plugins and options which may affect your projects. Here are some notable changes:
+TinyMCE has been upgraded up two major versions from 4 to 6. The API for [`HTMLEditorConfig`](api:SilverStripe\Forms\HTMLEditor\HTMLEditorConfig) and [`TinyMCEConfig`](api:SilverStripe\Forms\HTMLEditor\TinyMCEConfig) haven't changed at all, but there have been some changes to plugins and options which may affect your projects. Here are some notable changes:
 
-- The `styleselect` option for selecting a css style has changed to `styles`
-- The `spellchecker` plugin is now a premium "[Spell Checker Pro](https://www.tiny.cloud/docs/tinymce/6/introduction-to-tiny-spellchecker/)" plugin through TinyMCE cloud (though the `browser_spellcheck` options still works and is enabled by default)
-- The "paste from Microsoft Word" functionality is now part of a premium "[PowerPaste](https://www.tiny.cloud/docs/tinymce/6/introduction-to-powerpaste/)" plugin through TinyMCE cloud
-  - There is an [open call for maintainers](https://github.com/tinymce/tinymce/discussions/7487) for converting the old functionality into a new open source plugin
```

Comment:
This seems to be a somewhat pointless thing to point out here.

Reply:
These are changes that, if not pointed out, are likely to trip people up when upgrading. Having custom tinymce config is pretty common, so pointing out changes in that config which are likely to trip people up seems like exactly the sort of thing we should include

Reply:
I disagree - I think it's worth mentioning, as it is in my prior experience a commonly requested feature. It's worth pointing out that while the old plugin isn't available there is an option to update and maintain it themselves.
Clear upgrade path sometimes means just giving people options, and in this case there's this option which is an alternative to using the pro plugin (which we haven't even tried ourselves and don't know if it's fully compatible with the way we're loading the wysiwyg).

Reply:
restored